packages/grafana-ui/.storybook/webpack.config.js

```diff
@@ -78,12 +78,36 @@ module.exports = ({ config, mode }) => {
 
   config.optimization = {
     nodeEnv: 'production',
+    moduleIds: 'hashed',
+    runtimeChunk: 'single',
+    splitChunks: {
+      chunks: 'all',
+      minChunks: 1,
+      cacheGroups: {
```

Comment:
Not too read up on cacheGroups. Will this also be applied to dev enviornment, since nodeEnv is set to production? I tried to both build and run Storybook, it seemed to work fine.

Reply:
Great to hear that!